components/TactileItemsRow.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <tr>
+    <td class="title">{{ item.meta.title }}</td>
+    <td>{{ item.meta.format.split(' ')[1] }}</td>
```

Comment:
Ugly, better replace "Eine Reportage" with "Reportage"

Reply:
Just for the record: With the current setup, additional changes are required to solve this in an ideal way, i. e. rethinking the data we store and its structure.